File: 프로그래머스/lv0/120886. A로 B 만들기/A로 B 만들기.cpp
#include <string>
#include <vector>
#include <algorithm>

using namespace std;

int solution(string before, string after)
{
    sort(before.begin(), before.end());
    sort(after.begin(), after.end());

    if (before.length() == after.length() && before == after)
        return 1;
    else
        return 0;



    return -1;
}

File: 프로그래머스/0/120886. A로 B 만들기/A로 B 만들기.java
import java.util.stream.Collectors;

class Solution {
    public int solution(String before, String after) {
        String bef = before.chars()
                .mapToObj(i -> (" " + i)).sorted()
                .collect(Collectors.joining());

        String aft = after.chars()
                .mapToObj(i -> (" " + i)).sorted()
                .collect(Collectors.joining());

        return bef.equals(aft) ? 1 : 0;
    }
}